perf/src/sigverify.rs

```diff
@@ -234,20 +245,88 @@ fn do_get_packet_offsets(
         u32::try_from(sig_start)?,
         u32::try_from(msg_start)?,
         u32::try_from(pubkey_start)?,
+        u32::try_from(pubkey_len)?,
     ))
 }
 
-fn get_packet_offsets(packet: &Packet, current_offset: usize) -> PacketOffsets {
+fn get_packet_offsets(packet: &mut Packet, current_offset: usize) -> PacketOffsets {
     let unsanitized_packet_offsets = do_get_packet_offsets(packet, current_offset);
     if let Ok(offsets) = unsanitized_packet_offsets {
+        check_for_simple_vote_transaction(packet, &offsets, current_offset).ok();
         offsets
     } else {
         // force sigverify to fail by returning zeros
-        PacketOffsets::new(0, 0, 0, 0)
+        PacketOffsets::new(0, 0, 0, 0, 0)
     }
 }
 
-pub fn generate_offsets(batches: &[Packets], recycler: &Recycler<TxOffset>) -> TxOffsets {
+fn check_for_simple_vote_transaction(
+    packet: &mut Packet,
+    packet_offsets: &PacketOffsets,
+    current_offset: usize,
+) -> Result<(), PacketError> {
+    if packet_offsets.sig_len != 1 {
```

Comment:
FYI (+ @sakridge) there are at least two validators on mainnet that use 2 sig votes:
- https://explorer.solana.com/address/p2pUJruVeKTuZ41zbGWcxienmzRFnUa9fjBHigsR3QC
- https://explorer.solana.com/address/DWvDTSh3qfn88UoQTEKRV2JnLt5jtJAVoiCo3ivtMwXP

I think it would be better to remove this restriction

Reply:
feels like checking number of sigs is a quick way to rule out vote, if it is not 100%, might as well remove this check. Unless `sig_len==1 || sig_len == 2` is all there is. 
BTW, how do I see the 2 sigs vote in via above links?

Reply:
Yea. `sig_len == 1 || sig_len == 2` should be fine, more than 2 doesn't make sense. Could be `sig_len <= 2` because I think we exclude `sig_len == 0` already in another place.

Reply:
> feels like checking number of sigs is a quick way to rule out vote, if it is not 100%, might as well remove this check. Unless `sig_len==1 || sig_len == 2` is all there is.
> BTW, how do I see the 2 sigs vote in via above links?

Any of those account's transactions have two account signers

Reply:
OK, putting in a new PR for this.